### PR TITLE
Adds css class to button elements

### PR DESCRIPTION
**Pull request by ezra-obiwale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ html-duration-picker.js is a very tiny JS library used for transforming a native
 
 ![Duration Picker Input](https://github.com/nadchif/html-duration-picker.js/blob/master/screenshots/default-after.jpg)
 
+**Hide Seconds** ```<input type="text" class="html-duration-picker" data-hide-seconds>```
```
Comment (nadchif) [inline README.md:22]: Please move this to [DOCUMENTATION.md](https://github.com/nadchif/html-duration-picker.js/blob/master/DOCUMENTATION.md)

File: src/html-duration-picker.js [hunk 2]
```diff
@@ -327,9 +327,11 @@ export default (function() {
       scrollUpBtn.setAttribute('aria-label', 'Increase duration');
       scrollUpBtn.setAttribute('style', `text-align:center; width: 16px;padding: 0px 4px; border:none; cursor:default;
         height:${(picker.offsetHeight/2)-1}px !important; position:absolute; top: 1px;`);
+      scrollUpBtn.classList.add('scroll-up');
```
Comment (nadchif) [inline src/html-duration-picker.js:330]: Please help me understand, what is the purpose or effect of this?

Comment (ezra-obiwale) [reply src/html-duration-picker.js:330]: Having the css class on the elements makes them customizable.

For instance, I needed to hide the buttons in my use case.

Comment (nadchif) [reply src/html-duration-picker.js:330]: I see, you're right! That's brilliant

**Comment by nadchif** [comment]
Thanks so much again 👏 

💡 Any chance you could do a `data-hide-buttons` or something like that? 😁

**Pull request merged**